WORKSPACE

```
@@ -292,7 +292,7 @@ nixpkgs_package(
     # a workaround for
     # https://github.com/bazelbuild/bazel/issues/2927.
     nix_file_content = """
-    with import <nixpkgs> {};
+    with import <nixpkgs> { config = {}; };
```

```suggestion
    with import <nixpkgs> { config = {}; overlays = []; };
```
`nixpkgs` also defaults the `overlays` attribute to files in `~/.config/nixpkgs`.